syntax_and_semantics/methods_and_instance_variables.md

```diff
@@ -7,13 +7,25 @@ class Person
   def initialize(@name : String)
     @age = 0
   end
+  
+  def age
```

Comment:
Maybe we should also write the `name` getter, what do you think? Mostly to comply with "ask for its name and for its age" in the paragraph below